test/shared/src/main/scala/zio/test/RunnableSpec.scala

```diff
@@ -17,10 +17,12 @@
 package zio.test
 
 import zio.UIO
+import zio.test.reflect.Reflect.EnableReflectiveInstantiation
 
 /**
  * A `RunnableSpec` has a main function and can be run by the JVM / Scala.js.
  */
+@EnableReflectiveInstantiation
```

Comment:
This makes ZIO test have a dependency on portable reflection library. Hmm, we have to think if it's worth it or not. Does it support Dotty?

Reply:
Looks like it might not...
It says: `Scala 2.10.x, 2.11.x, 2.12.x and 2.13.0-M4` on their [readme](https://github.com/portable-scala/portable-scala-reflect#setup).
In any case it looks like I'm still missing some pieces to have sbt run specs under Scala.js

Reply:
Scalatest recently ported & published for dotty, might want to look at what they do. (Also, dotty works just fine with 2.12 test frameworks under DottyCompat mode, might not need any special support)

Reply:
Well, let's leave it in for now, we can decide for sure later! Maybe some small Scala.js code is all we need to do the reflection (iterate keys in `global` / `window`).